corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -203,8 +208,14 @@ def get_form_data_schema(request, domain, form_unique_id):
         data.append(get_session_schema(form))
         if form.requires_case() or is_usercase_in_use(domain):
             data.append(get_casedb_schema(form))
-    except Exception:
-        logger.exception("schema error")
+    except AppManagerException as e:
+        notify_exception(request, message=e.message)
+        return HttpResponseBadRequest(_(
+            "There is an error in the case management of your application. "
+            "Please fix the error to see case properties in this tree"
```

Comment:
I think that's a great idea, but it will require a vellum change. It's only set up to have text come back instead of html. This would be fine for this error (which is just text without a link) but there are some errors that come back that would include `<a href='your form'>form 1</a>` which would not be very helpful